tools/cldr-apps/src/main/webapp/js/new/cldrAjax.js

```diff
@@ -8,7 +8,7 @@
  * and to hide everything else, minimizing global scope pollution.
  */
 const cldrAjax = (function () {
-  const ST_AJAX_DEBUG = false;
+  const ST_AJAX_DEBUG = true;
```

Comment:
should this stay on by default?

Reply:
it makes the console a bit noisy, so maybe we should just turn it on and off as seems most useful; I didn't notice I had switched it in this PR